lib/mintacoin/blockchains/blockchain_txs.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Mintacoin.BlockchainTxs do
+  @moduledoc """
+  This module is the responsible for the CRUD operations for blockchains transactions and also for the aggregate operations within the blockchain context.
+  """
+  import Ecto.Query
+
+  alias Ecto.{Changeset, Query.CastError, UUID}
+  alias Mintacoin.{BlockchainTx, Repo, Wallet}
+
+  @type id :: UUID.t()
+  @type wallet_id :: Wallet.t()
+  @type tx_id :: String.t()
+  @type changes :: map()
+  @type parameter :: keyword()
+  @type error :: Changeset.t() | :invalid_params | :not_found
+
+  @spec create(changes :: changes()) :: {:ok, BlockchainTx.t()} | {:error, error()}
+  def create(changes) when is_map(changes) do
+    %BlockchainTx{}
+    |> BlockchainTx.create_changeset(changes)
+    |> Repo.insert()
+  end
+
+  def create(_changes), do: {:error, :invalid_params}
+
+  @spec update(id :: id(), changes :: changes()) :: {:ok, BlockchainTx.t()} | {:error, error()}
+  def update(id, changes) when is_binary(id) and is_map(changes) do
+    case Repo.get(BlockchainTx, id) do
+      blockchain_tx when not is_nil(blockchain_tx) ->
+        blockchain_tx
+        |> BlockchainTx.changeset(changes)
+        |> Repo.update()
+
+      nil ->
+        {:error, :not_found}
+    end
+  end
```

Comment:
@jhonatan-kmt I think here to make the function look cleaner we can do this:
```suggestion
def update(id, changes) when is_binary(id) and is_map(changes) do
    BlockchainTx
    |> Repo.get(id)
    |> BlockchainTx.changeset(changes)
    |> Repo.update()
rescue
    FunctionClauseError -> {:error, :not_found}
end
```